citadel_types/src/crypto/mod.rs

```diff
@@ -351,8 +351,8 @@ pub enum KemAlgorithm {
 )]
 pub enum SigAlgorithm {
     #[default]
-    None = 0,
-    Falcon1024 = 1,
+    Falcon1024 = 0,
```

Comment:
Keep `None` as 0, just shift the default tag down to Falcon. This keeps the encoding consistent between protocol versions, so there are no surprises. Otherwise, we would need to update the protocol versions.